imod/mf6/hfb.py

```diff
@@ -403,6 +403,14 @@ def to_mf6_pkg(
 
         return Mf6HorizontalFlowBarrier(**barrier_dataset)
 
+    def is_empty(self) -> bool:
+        if super(__class__, self).is_empty():
```

Comment:
if the hfb object has linestrings, then the baseclass.is_empty() will return False, because the Geometry object is in there, and it conforms the EmptyIndexesSchema. 
That's why after asking the base class, we check if all the linestrings are empty. If they are, then hfb.is_empty()  returns True.